packages/schema-generator/src/generate_schema.ts

```diff
@@ -19,7 +20,10 @@ export class SchemaGenerator {
     const dbConfig = parseDatabaseUrl(props.connectionUri);
 
     try {
-      const schema = await TypescriptDataSchemaGenerator.generate(dbConfig);
+      const schema = await TypescriptDataSchemaGenerator.generate({
+        ...dbConfig,
+        connectionUriSecretName: props.secretName,
```

Comment:
Is this the Amplify secret name or the SSM path to the connection URI? Since it's calling the low level API used by both Gen2 and GraphQL CDK construct users, I assume it is the later?

Reply:
It is the amplify secret name. Here, the argument `connectionUriSecretName` is just a string, we use this value in the generated file as [shown in this snapshot](https://github.com/aws-amplify/amplify-category-api/blob/2c8ee6b16c1bb2b5dececa70354d7f4fff616497/packages/amplify-graphql-schema-generator/src/__tests__/__snapshots__/generate-ts-data-schema.test.ts.snap#L115) (which returns a value of type `BackendSecret`). Later when we wire `defineData` with the CDK construct, we need to invoke `BackendSecret.resolvePath` to pass the actual SSM path to the CDK construct.

Reply:
Thanks for clarifying.